Details addition (#5934)
MicrosoftDocs/windows-itpro-docs: windows/security/threat-protection/microsoft-defender-atp/enable-controlled-folders.md

>     * **Enable** - Malicious and suspicious apps will not be allowed to make changes to files in protected folders. A notification will be provided in the Windows event log

Reviewer: Cosmetic / codestyle: (by previous contributors) missing ending period dot.
```suggestion
    * **Enable** - Malicious and suspicious apps will not be allowed to make changes to files in protected folders. A notification will be provided in the Windows event log.
```

    * **Disable (Default)** - The Controlled folder access feature will not work. All apps can make changes to files in protected folders.
    * **Audit Mode** - If a malicious or suspicious app attempts to make a change to a file in a protected folder, the change will be allowed but will be recorded in the Windows event log. This allows you to assess the impact of this feature on your organization.

>     * **Block disk modification only** - Attempts by untrusted apps to write to disk sectors. The Windows event log will record these attempts under Applications and Services Logs > Microsoft > Windows > Windows Defender > Operational > ID 1123.

Reviewer: I think we need to modify this text a bit, to turn it into normal readable text. I will start by making a suggestion:
```suggestion
    * **Block disk modification only** - Attempts by untrusted apps to write to disk sectors will be logged in Windows Event log. These logs can be found in **Applications and Services Logs** > Microsoft > Windows > Windows Defender > Operational > ID 1123.
```
@JohanFreelancer9 : I am not quite sure about my suggestion, maybe you could suggest a better phrasing.

>     * **Block disk modification only** - Attempts by untrusted apps to write to disk sectors. The Windows event log will record these attempts under Applications and Services Logs > Microsoft > Windows > Windows Defender > Operational > ID 1123.

Reviewer 2: ```suggestion
    * **Block disk modification only** - Attempts by untrusted apps to write to disk sectors. The Windows event log will record these attempts under _Applications and Services Logs_ > **Microsoft** > **Windows** > **Windows Defender** > **Operational** > **ID 1123**.
```

>     * **Block disk modification only** - Attempts by untrusted apps to write to disk sectors. The Windows event log will record these attempts under Applications and Services Logs > Microsoft > Windows > Windows Defender > Operational > ID 1123.

Reviewer 2: ```suggestion
    * **Block disk modification only** - Attempts by untrusted apps to write to disk sectors will be logged in Windows Event log. These logs can be found in **Applications and Services Logs** > **Microsoft** > **Windows** > **Windows Defender** > **Operational** > **ID 1123**.
```

>     * **Audit disk modification only** - Only attempts to write to protected disk sectors will be recorded in the Windows event log (under Applications and Services Logs > Microsoft > Windows > Windows Defender > Operational > ID 1124). Attempts to modify or delete files in protected folders will not be recorded.

Reviewer 2: ```suggestion
    * **Audit disk modification only** - Only attempts to write to protected disk sectors will be recorded in the Windows event log (under **Applications and Services Logs** > **Microsoft** > **Windows** > **Windows Defender** > **Operational** > **ID 1124**). Attempts to modify or delete files in protected folders will not be recorded.
```

>       ![Screenshot of group policy option with Enabled and then Enable selected in the drop-down](../images/cfa-gp-enable.png)

Reviewer: I know this line is not part of your changes in this PR, but it would be useful to see this "Alt text" match the actual selections in the image:
```suggestion
      ![Screenshot of the group policy option Enabled and Audit Mode selected in the drop-down](../images/cfa-gp-enable.png)
```
